api/src/catalog/catalog.utils.ts

```diff
@@ -158,4 +220,18 @@ export async function fetchCatalog(overwrite: boolean) {
     ),
   );
   await Promise.all(processSeasons);
+
+  for (const seasonCode of seasons) {
```

Comment:
Don't use await in a for loop if the loop isn't meant to be sequential. Parallelize it with `Promise.all`.

Also, try to reuse the data returned by `fetchData`—no need to fetch it again in this function. Some refactoring is probably necessary (for example you may want to actually do season sitemap generation as part of `fetchData`.

Reply:
thank u good call
should be better